src/features/chat/handlers.ts

```
@@ -9,46 +9,47 @@
+    isCodeBlock,
+    codeBlockText,
+    sentences,
+    aiTextLog,
+    currentSlideMessages,
+  } = options
+
+  // タグと返答を分離
   const tagMatch = receivedMessage.match(/^\[(.*?)\]/)
   if (tagMatch && tagMatch[0]) {
     tag = tagMatch[0]
     receivedMessage = receivedMessage.slice(tag.length)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)